websites/D/Discords.com/presence.ts

```diff
@@ -395,7 +395,7 @@ presence.on("UpdateData", async () => {
 		presenceData.details = document
 			.querySelector("div.bot-title-bp")
 			?.querySelector("h2")?.textContent;
-		presenceData.state = `💻 ${stats[0]?.textContent?.replace(
+		presenceData.state = `💻 ${stats.?at(0)?.textContent?.replace(
```

Comment:
?.at(0)
^
oops

Reply:
Hmm for some reason it wouldn't let me compile it. I know it's a type issue, I just can't remember what type the .at() property comes under.

Reply:
[`.at`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) is a method on arrays. Another way that would also have worked would have been:
```js
stats?.[0]
```